### ISSUE TITLE
`@id` should imply `@unique` for foreign keys

### ISSUE THREAD

**Issue by he-la** [zenstackhq/zenstack#392]
**Description and expected behavior**
```
model M1 {
  m2_id    String @id
  m2       M2      @relation(fields: [m2_id], references: [id])
}

model M2 {
  id       String @id

  m1       M1?
}
```

errors with

```
Validation errors:
line 68: Value is not assignable to parameter []
line 82: Field "m2_id" is part of a one-to-one relation and must be marked as @unique or be part of a model-level @@unique attribute []
```

**Environment (please complete the following information):**

-   ZenStack version: `1.0.0-alpha.113`
-   Prisma version: `4.11.0`
-   Database type: Postgresql

**Comment by ymc9** [comment]
Hi @he-la , this issue is resolved by the latest alpha.114 release.

**Comment by ymc9** [comment]
Fixed by #400
